packages/frontend/src/hooks/useCrabPosition.ts

```diff
@@ -25,6 +25,10 @@ export const useCrabPosition = (user: string) => {
   const [minPnlUsd, setMinPnlUsd] = useState(BIG_ZERO)
   const [minPnL, setMinPnL] = useState(BIG_ZERO)
 
+  const depositedDepedancy = data
+    ?.map((item) => `${item.ethAmount.toString()}:${item.lpAmount.toString()}:${item.ethUsdValue.toString()}`)
+    .join(' ')
```

Comment:
can we use data.length as the dependency instead or is this a better dependency?

Reply:
The issue is happening because the calculation isn't handled even after the updated data. Are we sure that there's no case for having updated data (ethAmount or ethUsdValue) though there's no length change?

Reply:
This is actually a nice idea IMO! History will not change but I think this is the best idea for other cases like changing to a different address with the same length!